src/main/java/com/vg/hm/samples/usermanagementsvc/service/VehicleService.java

```diff
@@ -0,0 +1,21 @@
+package com.vg.hm.samples.usermanagementsvc.service;
+
+import com.vg.hm.samples.usermanagementsvc.service.model.Make;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+import java.util.List;
+
+public interface VehicleService {
+    /**
+     * Gets all car makes
+     * @return list of Make objects
+     */
+    public List<Make> getMakes() throws IOException;
+
+    /**
+     * Gets a list of predefined makes
+     * @return
+     */
+    public List<Make> getDummyMakes();
```

Comment:
If this is only called by the fallback, then you don't need to include this on the interface. Your implementation can talk directly to the private method, unless you need to use this method somewhere else.

Reply:
Ticket #3 calls for a dummy endpoint with hardcoded makes That's why this method is included in the interface so it can be called from the REST service..